src/bci_build/package/base/README.md.j2

```diff
@@ -5,24 +5,69 @@
 
 SUSE Linux Enterprise Base Container Images (SLE BCI) provide truly open,
 flexible, and secure container images and application development tools. The
-images consist of container environments based on SUSE Linux Enterprise and
-designed to be a secure base for any containerized workload.
+images consist of container environments based on SUSE Linux Enterprise Server
+and are designed to provide a secure base for any containerized workload.
 
 SLE BCI is freely available, re-distributable, and supported across many
-different environments. These templates and tools address modern, containerized
-application development and CI/CD application containerization.  They can be
+different environments. These container images address modern, containerized
+application development and CI/CD application containerization. They can be
 used immediately by developers and integrators without the “lock-in” imposed by
 other offerings.
 
 SLE BCI inherits industry-leading security and compliance from SUSE Linux
-within your container build process. The container images are designed to be a
-secure base for any application workload. SUSE ensures that compliance
-standards are applied consistently and continuously improves security-related
-capabilities.
-
-SLE BCI is lightweight and easy to adopt, with the ability to run with any
-Linux OS. Avoid lock-in imposed by other vendors and get exactly what you need,
-fast. SLE BCI delivers a flexible developer experience that accounts for,
-integrates with, and supports language-native tools and workflows.
+Enterprise within your container build process. The container images are
+designed to provide a secure base for any application workload. SUSE ensures
+that compliance standards are applied consistently and continuously improves
+security-related capabilities.
+
+SLE BCI is lightweight and easy to adopt, with the ability to run using any OCI
+compatible container runtime. Avoid lock-in imposed by other vendors and get
+exactly what you need. SLE BCI delivers a flexible developer experience
+that accounts for, integrates with, and supports language-native tools and
+workflows.
+
+
+## Usage
+
+This container image ships the `zypper` package manager, the free `SLE_BCI`
+repository and the `container-suseconnect` utility, which can give you access to
```

Comment:
This sounds a bit better:

which can give you access > which can provide you with access